app/web-app/src/blockchain/deployments/production/fairFund_deployment.json

```diff
@@ -1 +1 @@
-{"fairFund":"0x1BADd20Eed4f56DA23A3544c3c373B18024D8Bd6"}
```

Comment:
This address should not be changed, as it is testnet deployment address, please do not include this file in the commit.